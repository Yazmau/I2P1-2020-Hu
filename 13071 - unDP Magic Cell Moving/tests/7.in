1000
1 1 344512164
4
3 905088515
4
1 1 831147244
1 3 607821938
2 2
4
1 0 694706520
4
2 0
3 57482590
3 799613524
2 1
4
4
4
2 0
2 0
2 3
2 2
1 0 1531503
4
2 3
2 0
1 1 83734933
2 1
1 2 792097012
1 2 830765952
1 3 947210091
3 452174873
4
1 2 18355219
1 3 585010118
1 0 588954555
3 215373980
1 0 853054998
1 3 808319188
3 349249770
2 2
1 3 55406634
3 463833704
4
4
2 2
1 3 606946629
1 1 866155555
4
4
3 827268945
4
1 1 76525027
1 0 133811328
4
4
4
2 1
4
1 1 388604797
2 2
1 0 985713619
4
4
4
1 3 869780190
2 1
2 3
2 0
3 183104921
3 896364821
1 2 867590431
3 640768532
4
4
4
2 1
4
4
1 2 616115586
1 1 864062430
2 1
1 0 692115217
4
2 0
2 0
3 559942478
2 3
2 0
4
2 0
4
3 787034904
1 1 721359568
2 2
3 187099727
1 1 514882513
1 1 537611849
1 3 752193964
3 678242579
3 621350840
4
4
2 1
1 2 70572490
4
1 3 169155502
2 3
2 1
3 986983659
2 2
1 2 282513435
2 1
2 3
1 1 63154618
3 117871554
2 2
3 410741023
3 175199173
3 831937475
1 2 582019728
1 0 782884369
4
2 2
4
1 0 717266049
1 0 505075212
1 1 147655414
4
2 0
1 1 624096916
2 0
2 3
1 2 408959830
1 1 664466733
4
2 2
2 1
2 1
3 984176443
2 3
1 1 15798791
3 470478486
3 671961584
1 1 183944061
1 2 181753215
1 2 174936234
2 1
1 1 310350270
4
2 2
3 830925264
4
2 0
1 2 877654415
4
2 3
2 1
4
3 554442180
3 446216779
1 0 227241613
3 191132993
2 0
3 456300291
4
3 533442038
2 2
3 903585552
2 1
1 2 419421646
3 924656064
4
1 1 409422210
2 3
2 2
1 3 856577819
4
1 0 324469691
2 0
2 1
3 365425948
1 0 640871253
3 628272259
4
1 1 714109787
2 2
2 3
4
2 0
1 2 286235873
2 2
1 3 545297912
2 0
3 374303567
4
2 2
4
4
4
1 0 361683165
4
4
2 2
2 2
2 3
2 0
2 0
1 3 236884534
1 2 80490991
1 2 29145480
3 877657095
4
4
1 1 344404920
4
3 228156047
3 548614393
3 173943356
2 3
3 165100181
2 1
3 375112990
4
4
1 3 636102851
2 3
3 525531157
4
3 835481790
4
2 2
4
2 1
4
4
2 1
4
2 2
3 794852693
2 2
3 832715075
3 391150690
4
2 3
2 2
2 2
2 1
1 0 756088051
2 3
4
2 0
4
1 3 895031515
1 1 307267511
3 949787646
1 3 548044691
1 2 794227846
1 1 552228747
2 3
2 2
1 3 324708109
2 1
3 981032653
4
2 0
4
4
1 0 841117064
3 724658588
1 1 987721787
4
1 0 317649896
2 3
2 1
1 2 554515904
1 2 288792746
4
3 906450152
1 3 871070249
4
2 2
4
1 2 725658843
2 3
3 271592538
4
4
2 3
3 194784808
4
4
2 2
3 162408482
4
4
2 0
2 3
4
3 521646652
2 0
1 0 587186079
1 2 961533969
2 2
3 198361136
2 3
3 488734318
1 0 390295228
4
4
1 2 967416295
4
4
4
2 0
1 0 98174418
3 599721601
3 685171795
4
1 0 641773318
4
4
2 3
4
3 357127984
4
4
4
3 913411276
1 2 882546792
2 2
4
3 615690066
4
3 685186518
1 3 543948800
3 56937615
2 2
2 3
1 2 530651195
2 0
4
1 2 10213041
4
3 471898406
1 2 757299325
1 1 961872797
1 1 426161033
2 1
4
3 88651261
3 371627922
4
1 1 83514038
2 2
2 0
1 1 65984877
3 540687248
4
3 575937854
4
4
1 0 681252650
3 716783486
1 0 621346081
2 3
2 3
2 0
1 0 24424708
1 1 773646821
3 383717380
1 0 739292363
4
4
3 586149988
3 455017778
3 839403579
3 944135857
3 916798824
2 1
4
1 1 418746172
1 0 493273007
1 1 531064311
3 763994196
2 2
3 345971271
1 2 464495451
3 161673185
4
3 412602879
3 7881066
1 1 314471554
3 913370105
2 2
4
1 0 123884912
2 0
4
4
3 333783721
3 265997030
3 963663836
2 2
1 1 100434233
3 916627992
4
1 2 107489322
1 1 809148633
2 3
2 2
1 0 278849533
2 2
3 174463556
4
4
2 3
2 1
2 2
2 2
2 0
1 1 589725087
1 1 192437188
2 0
1 0 667386657
1 0 697961311
2 1
3 91529156
2 3
1 0 832080502
1 3 976002785
4
4
2 2
4
2 3
1 1 608151269
1 1 897407149
3 192734285
2 2
3 831375010
2 0
1 0 901927176
2 0
3 447351065
3 709794637
3 176801298
2 3
3 537840289
4
3 208336039
4
3 701981622
2 0
4
1 2 950566728
2 0
4
1 0 574239853
4
2 3
2 1
4
3 748324623
3 799221238
2 2
1 0 874723723
3 731411280
3 659369133
1 2 804764735
1 1 292921252
3 347165567
1 0 515903141
2 0
4
4
3 236130584
2 1
1 1 812822539
1 2 631947752
1 3 102794597
2 0
4
1 1 741053360
2 2
2 2
3 601055250
1 2 455849563
4
2 1
1 2 192071146
4
3 929096304
4
4
4
2 1
2 0
1 3 998942825
1 0 849062825
3 100070382
2 0
4
1 2 594934631
3 791718868
4
2 0
1 3 986284076
3 682480415
3 457841220
4
2 0
3 4799357
2 0
2 2
1 2 834003107
1 2 248648267
3 915180392
3 602284986
3 243968956
3 580021792
1 0 851068262
3 588040313
3 845209680
1 0 780689435
4
2 2
4
2 2
3 692697873
4
2 0
4
3 403990084
1 2 36399206
1 0 363334036
2 2
4
2 3
3 724163731
1 3 906544643
4
4
4
4
3 212565428
1 1 918944199
2 2
3 325419580
4
2 3
2 3
1 3 957206166
1 0 747078559
2 0
2 3
3 444017479
2 2
3 475397626
4
4
1 2 433101270
2 2
3 316143173
1 3 812532898
4
4
2 2
4
4
2 1
3 396487890
3 476901082
3 748543646
2 3
4
4
1 0 124718066
4
3 39872821
1 2 216933274
1 1 582103178
1 3 386004427
2 1
3 268917736
1 0 681603889
4
1 2 407332922
4
1 0 696730744
1 1 30500509
4
2 1
4
3 663268378
1 3 1034200
2 3
1 3 872166804
1 0 671175725
1 2 156758196
3 351691685
4
3 994732319
4
2 1
3 430381543
1 1 199287168
4
1 3 418924432
2 0
1 1 168692648
1 3 243984923
2 0
3 806419818
2 2
2 3
1 0 717393082
4
3 985707517
3 398801040
1 1 311417408
4
1 1 754276200
4
1 1 326956578
4
1 3 50704701
3 201393638
2 3
2 2
4
3 242353521
4
2 3
1 1 130920022
3 126221505
4
4
1 1 15046092
3 646150699
4
4
1 0 27881662
3 732714174
4
3 574690428
1 0 212703733
3 625814550
3 373166529
2 0
1 2 367780623
4
2 2
2 2
3 490984825
1 1 964980128
2 1
4
4
1 3 516318269
1 3 884596211
4
4
4
2 1
3 166408377
3 489488627
2 1
4
2 0
2 2
4
1 1 434449446
2 1
1 2 968798661
2 2
4
3 123766742
2 2
1 2 598027003
4
3 65918753
2 3
3 241763290
3 313696569
1 1 566546804
4
1 0 913754409
1 3 94384002
1 2 105938258
4
2 3
3 14210673
1 2 713161001
2 3
2 0
4
1 3 49093477
1 3 76711276
1 2 173294587
4
4
1 2 382123903
2 2
2 2
2 3
2 2
4
3 665779620
2 2
3 830527939
3 492281811
2 0
3 876688402
3 643600598
3 898227999
4
4
3 685905687
4
4
3 672215082
2 0
2 2
4
4
2 1
1 3 265908966
4
4
1 2 758530524
2 1
2 1
1 3 480420245
2 2
1 0 56269812
3 636691341
3 187090411
4
1 3 296433950
4
3 327624108
2 1
3 355909712
2 3
1 1 913920794
2 3
4
1 0 570433628
2 1
2 1
3 206882597
4
4
2 1
2 1
1 2 853532137
3 765006206
2 1
3 763530068
2 2
3 423751168
1 1 764957689
1 3 715358965
4
2 2
4
1 3 446495915
1 0 57972189
1 1 532366963
2 3
4
1 3 840592765
2 0
1 1 391051686
2 2
1 3 775477780
1 0 598362272
4
2 1
1 1 522612235
2 0
2 1
2 1
3 163359560
1 0 188855153
3 259168164
1 3 311019547
4
4
2 3
3 195263550
3 625234676
4
3 878519326
1 1 204152382
3 9661967
1 0 735623884
2 3
2 3
1 3 238414808
3 361177614
4
3 22364193
3 173685773
1 2 32249372
2 2
3 403599478
3 526105462
4
3 306464566
1 1 261356656
4
4
1 1 129616846
4
2 2
4
4
1 3 854879462
4
1 1 332047425
1 1 911447272
2 2
3 399408903
4
2 3
4
1 3 939876204
3 120486755
1 1 227259722
4
2 1
1 0 362597884
3 58013429
4
1 3 81327564
1 0 656653957
3 980998334
2 0
3 235323098
3 530011168
1 3 116523992
3 970370324
3 949285298
4
4
2 2
2 1
4
1 2 191035670
1 0 405079850
4
4
3 789414730
1 0 400670341
1 2 726592745
2 3
1 0 803514478
2 0
3 534126158
4
2 0
4
4
2 3
2 3
1 1 204565169
1 3 770855765
1 0 226288476
1 2 232701639
3 223816436
1 2 494319814
3 715573932
1 3 921289760
3 766810241
3 541180588
3 27218600
1 0 313214906
2 2
1 1 293652518
3 318547671
3 964906450
3 36558174
4
3 738063923
3 385588064
4
2 2
1 1 558837537
2 1
3 109485787
3 413390576
4
4
3 316983816
2 0
1 3 877786177
1 3 912507011
1 2 198498807
1 1 207689778
2 0
2 3
3 541757600
1 0 150934081
4
2 1
2 3
1 1 738423479
2 1
3 71840904
1 3 921477403
4
3 738156694
2 3
4
2 1
2 0
2 0
1 2 730088722
3 433406359
1 2 420632184
2 0
4
3 436665956
4
4
3 444727982
1 1 468890340
1 1 667125414
4
1 2 6853126
2 2
1 1 291408882
4
4
1 2 705597927
2 0
3 977440483
2 3
4
1 0 979125052
3 645527251
1 0 535331099
3 941250076
3 55739296
3 127859315
2 1
2 3
4
1 3 36173907
3 801222046
2 1
4
4
2 1
1 2 976718462
2 3
2 0
3 736011288
4
3 121712181
1 2 165023967
3 821484322
2 1
3 15387200
3 610257261
2 1
2 1
1 0 92058123
2 0
4
2 0
2 1
4
3 521641759
4
2 3
3 566839650
3 697249902
4
3 409625584
3 3335279
4
1 0 816811129
3 453152439
4
3 488917767
1 2 963585076
4
4
3 423989710
1 1 149843333
2 2
4
2 2
4
2 3
3 689852533
2 0
1 0 397087785
