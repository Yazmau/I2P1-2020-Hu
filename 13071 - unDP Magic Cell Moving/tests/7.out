0
905088515
Invalid
Invalid
905088515
905088515
Invalid
799613524
799613524
799613524
Invalid
Invalid
Invalid
Invalid
905088515
Invalid
Invalid
Invalid
Invalid
452174873
Invalid
Invalid
Invalid
Invalid
Invalid
463833704
463833704
Invalid
Invalid
463833704
463833704
827268945
Invalid
Invalid
827268945
827268945
827268945
866155555
Invalid
866155555
866155555
866155555
Invalid
Invalid
640768532
640768532
640768532
Invalid
640768532
640768532
Invalid
864062430
Invalid
Invalid
Invalid
559942478
Invalid
559942478
Invalid
Invalid
621350840
621350840
514882513
Invalid
Invalid
Invalid
Invalid
831937475
582019728
Invalid
Invalid
582019728
Invalid
Invalid
Invalid
117871554
Invalid
Invalid
Invalid
Invalid
Invalid
984176443
Invalid
830925264
Invalid
830925264
830925264
456300291
924656064
877654415
Invalid
628272259
Invalid
Invalid
419421646
Invalid
374303567
419421646
419421646
419421646
Invalid
419421646
419421646
Invalid
Invalid
Invalid
Invalid
Invalid
877657095
877657095
Invalid
877657095
375112990
375112990
Invalid
525531157
835481790
Invalid
835481790
375112990
375112990
419421646
Invalid
Invalid
391150690
Invalid
Invalid
Invalid
Invalid
80490991
375112990
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
981032653
Invalid
981032653
981032653
Invalid
724658588
Invalid
Invalid
Invalid
724658588
Invalid
906450152
80490991
Invalid
271592538
271592538
Invalid
194784808
194784808
162408482
162408482
Invalid
194784808
Invalid
Invalid
Invalid
Invalid
488734318
488734318
Invalid
488734318
488734318
488734318
Invalid
685171795
Invalid
685171795
685171795
835481790
357127984
357127984
357127984
882546792
615690066
Invalid
Invalid
543948800
Invalid
543948800
Invalid
Invalid
Invalid
56937615
371627922
Invalid
Invalid
540687248
575937854
575937854
Invalid
Invalid
Invalid
Invalid
Invalid
383717380
383717380
716783486
Invalid
Invalid
Invalid
161673185
Invalid
464495451
Invalid
913370105
913370105
916627992
Invalid
Invalid
Invalid
Invalid
174463556
174463556
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
963663836
963663836
916627992
Invalid
Invalid
Invalid
Invalid
537840289
208336039
Invalid
701981622
Invalid
Invalid
701981622
701981622
Invalid
176801298
Invalid
Invalid
Invalid
Invalid
799221238
799221238
Invalid
Invalid
91529156
Invalid
Invalid
601055250
Invalid
Invalid
601055250
929096304
929096304
929096304
Invalid
Invalid
849062825
Invalid
791718868
Invalid
457841220
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
845209680
929096304
692697873
929096304
Invalid
Invalid
692697873
Invalid
724163731
724163731
724163731
724163731
Invalid
325419580
Invalid
Invalid
475397626
475397626
316143173
316143173
Invalid
316143173
316143173
Invalid
812532898
812532898
812532898
Invalid
Invalid
268917736
268917736
Invalid
268917736
30500509
Invalid
Invalid
Invalid
Invalid
351691685
994732319
430381543
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
994732319
Invalid
398801040
Invalid
398801040
Invalid
398801040
Invalid
216933274
242353521
Invalid
126221505
126221505
Invalid
646150699
646150699
Invalid
732714174
Invalid
Invalid
39872821
Invalid
Invalid
130920022
130920022
Invalid
Invalid
130920022
130920022
130920022
Invalid
Invalid
489488627
Invalid
Invalid
489488627
968798661
Invalid
123766742
Invalid
313696569
Invalid
Invalid
313696569
Invalid
Invalid
14210673
Invalid
Invalid
14210673
14210673
Invalid
Invalid
Invalid
Invalid
713161001
Invalid
898227999
898227999
685905687
685905687
Invalid
492281811
492281811
Invalid
492281811
492281811
Invalid
Invalid
Invalid
Invalid
187090411
187090411
Invalid
Invalid
Invalid
296433950
Invalid
206882597
206882597
Invalid
Invalid
Invalid
423751168
Invalid
423751168
Invalid
Invalid
Invalid
715358965
Invalid
Invalid
Invalid
Invalid
715358965
Invalid
Invalid
Invalid
Invalid
Invalid
259168164
259168164
625234676
Invalid
Invalid
Invalid
Invalid
361177614
526105462
306464566
306464566
Invalid
306464566
Invalid
306464566
306464566
306464566
Invalid
Invalid
Invalid
399408903
854879462
Invalid
120486755
Invalid
58013429
Invalid
Invalid
Invalid
949285298
949285298
261356656
261356656
261356656
Invalid
Invalid
Invalid
534126158
570433628
570433628
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
36558174
385588064
Invalid
413390576
413390576
Invalid
Invalid
Invalid
541757600
Invalid
Invalid
71840904
541757600
Invalid
Invalid
Invalid
Invalid
Invalid
433406359
436665956
436665956
Invalid
Invalid
444727982
Invalid
316983816
316983816
Invalid
541757600
Invalid
Invalid
127859315
977440483
977440483
Invalid
736011288
Invalid
Invalid
Invalid
92058123
Invalid
Invalid
92058123
521641759
Invalid
697249902
3335279
453152439
Invalid
488917767
488917767
610257261
976718462
Invalid
Invalid
